python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -1358,6 +1357,20 @@ def _get_task_parent_entity_type(self, task_id):
 
         return cache[self.TASK_PARENT_TYPES][task_id]
 
+    def _get_toolkit_manager(self):
+        """
+        Gets an initialized ToolkitManager object.
+
+        :returns: A ToolkitManager object.
+        """
+        if self.TOOLKIT_MANAGER is None:
```

Comment:
Why are we keeping this around? performance? I don't think we gain much in terms of performance so wondering if it's better to instantiate a new object every time?